sklearn/manifold/t_sne.py

```diff
@@ -767,10 +770,10 @@ def _fit(self, X, skip_num_points=0):
             pca = PCA(n_components=self.n_components, svd_solver='randomized',
                       random_state=random_state)
             X_embedded = pca.fit_transform(X)
-        elif isinstance(self.init, np.ndarray):
-            X_embedded = self.init
         elif self.init == 'random':
             X_embedded = None
+        elif isinstance(self.init, np.ndarray):
```

Comment:
My thinking was that in case any of the statements inside the conditional ever change, or any other string options are added to 'init' in the future, it's more readable to have all the string options in the same order first and then the NumPy array option. The order of the possible options seems to be 'pca', 'random', and then NumPy array in other places throughout the code/doc, so this change makes them consistent here.